docs/UserGuide.md

```diff
@@ -280,75 +334,92 @@ Example:
 
 #### 7.2.5. Sorting contacts : `sort`
 
-Sorts the list of contacts displayed by 1 or more parameter(s) chosen by you.
+Organising your contacts can make tracking and managing them easier, especially when you have lots of contacts. You can organise your contacts in the order you prefer using the `sort` command as shown below. Given below are the orders that you can choose each parameter to be sorted by.
 
 How *names (n/)*, *emails (e/)*, *addresses (a/ )* are sorted:
 * In alphabetical order. (e.g. `Al` comes before `Alfred` which comes before `Brad`)
-* Case insensitive. (e.g. `Al`, `al`, `AL`, and `aL` are identical when it comes to sorting)
+* Case-insensitive. (e.g. `Al`, `al`, `AL`, and `aL` are identical when it comes to sorting)
 
-How *phone numbers (p/)* are sorted: 
+How *phone numbers (p/)* are sorted:
 * In increasing numerical order. (e.g. `123` comes before `125` which comes before `1234`)
 
-How *tags (t/TAG)* are sorted:
+How *[tags](#tag) (t/TAG)* are sorted:
 * Contacts with the *TAG* you specified will come before contacts without the *TAG*.
 
-<div markdown="block" class="alert alert-info">
+<div markdown="block" class="alert alert-primary">
 
-**:information_source: Note:**<br>
+**:memo: Note:**<br>
 * When sorting by tags, unlike other parameters, you have to specify a value (an existing `TAG`) to sort by.
-* For other parameters (i.e. `n/ e/ a/ p/`), values provided are ignored. (e.g. sorting by `n/Alfred` is a valid command, the list will be sorted by name and the given value `Alfred` is ignored)
+* For other parameters (i.e. `n/ e/ a/ p/`), values provided are ignored. (e.g. sorting by `n/Alfred` is a valid command, the list will be sorted by name alphabetically, and the name given `Alfred` is ignored)
 
```

Comment:
```suggestion
* For other prefixes(i.e. `n/ e/ a/ p/`), parameters provided are ignored. (e.g. sorting by `n/Alfred` is a valid command, the list will be sorted by name alphabetically, and the name given `Alfred` is ignored)

```
I think we should include the prefix and parameter in our key definition.

Reply:
Prefixes and parameters are included in the key definition under PR #249 And my usage of the terms parameters and values also adhere to the definition there. (i.e. parameter = prefix + value)